include/room_member.h

```diff
@@ -0,0 +1,92 @@
+// Copyright 2017 Tunnler Project
+// Licensed under GPLv2 or any later version
+// Refer to the license.txt file included.
+//
+
+#pragma once
+
+#include <deque>
+#include <string>
+
+#include "room_meassage_types.h"
+#include "tunnler.h"
+
+#include "RakNetStatistics.h"
+#include "RakPeerInterface.h"
+
+class RoomMember final {
+public:
+
+	struct MemberInformation {
+		std::string nickname;
+		std::string game_name;
+		MacAddress mac_address;
+	};
+
+	enum class State {
+
+		//Actual states
+		StateIdle, // Default state
```

Comment:
Remove the "State" word from all of these, and remove the ones we won't be using in the near future